website/blog/2019-01-21-jest-24-new-website-babel-7-and-typescript-support.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Jest 24: New Website, Babel 7 and TypeScript support
+author: Simen Bekkhus
+authorURL: https://github.com/SimenB
+authorFBID: 100003004880942
+---
+
+Today we are happy to announce the next major release of Jest - version 24! It's been 4 months since the last release (and 8 since Jest 23), so this is a meaty one. Highlights include upgrading to Babel 7 across the board (bringing with it built-in support for testing TypeScript) fixing some long standing issues with missing console output and performance issues when computing large diffs, as well as a brand new sparkling website. ✨
```

Comment:
```suggestion
Today we are happy to announce the next major release of Jest - version 24! It's been 4 months since the last minor release (and 8 months since Jest 23), so this is a meaty one. Highlights include built-in support for TypeScript by upgrading the Jest internals to Babel 7, fixing some long standing issues with missing console output, performance issues when computing large diffs, and a brand new sparkling website. ✨
```

- Call out minor release
- Add unit to "8"
- Lead with typescript support, mentioning the babel upgrade as a parenthetical
- Fix list and add oxford comma 🤓

Reply:
https://github.com/facebook/jest/pull/7670/commits/20acd58214b8a07a887794432b6d940bbd30b7bb

ok?